adhocteam/Head-Start-TTADP
# Print styles for activity reports

**Description of change**

- Setup a '@media print' section in App.css with expected styles
- Add classes to components so they can be selected in CSS for print-related styles
- Create a PrintSummary component, place on ReviewSubmit page

**How to test**

Go to an Activity Summary, open your print menu. Take a look at the preview, print it out, save to PDF. You should see black text on a white background. You shouldn't see the sidebar menu or other UI elements not needed for a report.

View other pages as well.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/108

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
